.github/CONTRIBUTING.md

```diff
@@ -159,6 +160,47 @@ This can happen for several reasons:
 To exclude a task from validation, open the corresponding Jupyter notebook and choose ```View -> Cell Toolbar -> Tags``` to see and edit the tags for each cell. Add the tag that is the most fitting description of the failure cause to the cell. 
 After you are done with editing the notebook, choose ```View -> Cell Toolbar -> None``` to turn off tags editing view for the subsequent users of this notebook. Finally, save the notebook.
 
+#### Validate changes to `%kata` magic and `%check-kata` magic on local machine
+1. Add logging statements to the [`KataMagic`](./utilities/Microsoft.Quantum.Katas/KataMagic.cs) and [`CheckKataMagic`](./utilities/Microsoft.Quantum.Katas/CheckKataMagic.cs).
+   ```
+   Logger.LogDebug($"Value : {val}");
+   ```
+2. Generate a custom nuget package
+   1. Build the Microsoft.Quantum.Katas project to produce a NuGet package to get a version other than 1.0.0 by adding the following code in [this file](./utilities/Microsoft.Quantum.Katas/Microsoft.Quantum.Katas.csproj) by adding the following code
+   ```
+   <RootNameSpace>Microsoft.Quantum.Katas</RootNameSpace>
+   <Version>version-number</Version> ).
+   <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+   ```
+   2. Copy the generated .nupkg file to the folder in which your project resides (like PhaseEstimation kata) and remove the following line from your project's project file.
+   ```
+   <PackageReference Include="Microsoft.Quantum.Katas" Version="0.16.2104138035" />
```

Comment:
If we mention the package version here, we'll need to update version update script to cover this line. (You'll see that [right now](https://github.com/microsoft/QuantumKatas/blob/35c153fe5b4901b0a822813ddaa77d4aeedcf165/scripts/Update-QDKVersion.ps1#L94) we update another line in this script, but not this one). I'd suggest to avoid this (especially since here the version number can be pretty much anything) and rephrase this as "Remove the package reference to Microsoft.Quantum.Katas from your .csproj file"

Reply:
Done